### ISSUE TITLE
Support of wide

### ISSUE THREAD

**Issue by typiconman** [slavonic/cumd#3]
I think `<wide>` is not implemented correctly. Here is the current implementation:

```
font-size: 130%;
display: inline-block;
transform: scaleY(0.77);
```

But if we look at the printed Slavonic books, разрядка (what is the English term?) is just additional spacing between characters (see attached).

It is probably sufficient to do something like this:

```
letter-spacing: 0.25rem;
```

Also, I think the underlying MarkDown tag `+wide+` is not correctly implemented in the VS code preview panel on the right. It also displays as "big", rather than "wide".

![022v](https://user-images.githubusercontent.com/5428695/77347405-d6f9f600-6d48-11ea-978a-1322fc0916e9.png)

**Comment by pgmmpk** [comment]
Agree. The "stretching" does not look right. It makes characters look bold and also shifts them. Thank you for suggesting a better solution. Though to my eye `letter-spacing: 0.15rem` looks better...

**Comment by typiconman** [comment]
I think it can actually be different in different books, but I will need to look for some examples tomorrow.
But it's probably not that significant what value we set now. The end user can specify a different one.

**Comment by pgmmpk** [comment]
Right, the idea is that default CSS is just an example, and end user decides on the final look. Theoretically, one can use block-level parameters to control the amount of stretching, if needed. But feels like an overkill. It is not possible to reproduce exact down-to-the-letter  positioning anyways...

**Comment by pgmmpk** [comment]
Just released 1.2.1 that uses a better default CSS for the `<wide>`
